pom.xml

```diff
@@ -53,19 +53,23 @@
       <artifactId>junit-jupiter-engine</artifactId>
       <scope>test</scope>
     </dependency>
-
```

Comment:
I like the air, at least after test dependencies.

Reply:
I grouped test dependencies separately.